src/System.Private.Jit/src/Internal/Runtime/JitSupport/JitNodeFactory.cs

```diff
@@ -13,6 +13,11 @@
 
 namespace ILCompiler.DependencyAnalysis
 {
+    public class CompilationModuleGroup
```

Comment:
It might be better to just leave a non-virtual CanInline method on Compilation - the implementation would just forward to NodeFactory.CompilationGroup within the compiler.

It would avoid having to touch the JIT codebase - a CompilationModuleGroup class in the JIT codebase doesn't make much sense.

Reply:
Thanks, makes sense, fixed in 2nd commit.